### PR TITLE
[mds-policy-author] add container image build

### PR DESCRIPTION
**Pull request by oldschoolsysadmin**
Builds a docker container image for use in kubernetes

### CODE REVIEW

**Review by avatarneil — DISMISSED** [review]

File: container-images/mds-policy-author/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@aws-lambda/mds-policy-author",
```
Comment (mdurling) [inline container-images/mds-policy-author/package.json]: Package name/description is incorrect

Comment (avatarneil) [reply container-images/mds-policy-author/package.json]: It is..?

File: container-images/mds-policy-author/tsconfig.build.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.build.json",
+  "compilerOptions": {
+    "outDir": "dist"
+  },
+  "references": [
+    { "path": "../../aws-lambda/aws-serverless-express-handler/tsconfig.build.json" },
```
Comment (mdurling) [inline container-images/mds-policy-author/tsconfig.build.json]: This project reference looks incorrect.

Comment (mdurling) [reply container-images/mds-policy-author/tsconfig.build.json]: Should reference env-inject perhaps?

File: container-images/mds-policy-author/package.json (hunk 1 above)
Comment (avatarneil) [reply container-images/mds-policy-author/package.json]: It is indeed

File: container-images/mds-policy-author/index.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/*
```
Comment (mdurling) [inline container-images/mds-policy-author/index.ts]: Pretty sure this is the wrong `index` file ... should be an Express server; not a lambda handler.

**Review by mdurling — APPROVED** [review]
Assuming the container build issues are resolved @invertigo, this LGTM from a code perspective.

**Review by levi217 — APPROVED** [review]

File: packages/mds-policy/server.ts [hunk 4]
```diff
@@ -16,7 +16,7 @@ import { ApiServer } from '@mds-core/mds-api-server'
 import { api } from './api'
 
 const {
-  env: { npm_package_name, PORT = 4007 }
+  env: { npm_package_name, PORT = 4003 }
```
Comment (levi217) [inline packages/mds-policy/server.ts]: Interesting that these ports are being swapped.

Comment (mdurling) [reply packages/mds-policy/server.ts]: All the APIs have been setup to use distinct ports so you can run them locally concurrently. `mds-policy` has historically used port `4003`. `mds-policy-author` is assigned `4007`.

**Review by avatarneil — APPROVED** [review]

**Pull request merged**